cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBoxHelper.java

```diff
@@ -102,16 +102,6 @@ public void run() {
                 editBox.setTextColor(Color.WHITE);
                 editBox.setSingleLine();
                 editBox.setOpenGLViewScaleX(scaleX);
-                Resources r = mCocos2dxActivity.getResources();
```

Comment:
@zilongshanren  If I revert it, then It's not match with position of cocos label. Because I also removed the default padding from cocos label. So you think I revert it? please suggest what I do? I revert or keep it.